core/src/main/java/dk/alexandra/fresco/framework/util/DeterministicRandomBitGenerator.java

```diff
@@ -0,0 +1,37 @@
+package dk.alexandra.fresco.framework.util;
+
+/**
+ * <p>
+ * Java's SecureRandom is not 'deterministic' in the sense that calls to setSeed (at least using
```

Comment:
In general let's drop all the references to SecureRandom in the documentation of this interface. It is not longer a SecureRandom.